### PR TITLE
Card devexp_416 - Clean and update ruby image to follow recent changes in STI

### PR DESCRIPTION
**Pull request by soltysh**
This as long as other image updates (https://github.com/openshift/nodejs-0-10-centos/pull/3, https://github.com/openshift/wildfly-8-centos/pull/28) should be merged only after openshift/source-to-image#98 is merged and origin is updated with latest STI version.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -21,15 +19,16 @@ RUN yum install -y --enablerepo=centosplus epel-release \
...
 
 # Add configuration files, bashrc and other tweaks
-#
 ADD ./ruby /opt/ruby/
 ADD ./.sti/bin/usage /opt/ruby/bin/
 
+# Default STI scripts url
 ENV STI_SCRIPTS_URL https://raw.githubusercontent.com/openshift/ruby-20-centos/master/.sti/bin
+# Default destination of scripts and sources, this is where assemble will look for them
+ENV LOCATION /tmp
```
Comment (mfojtik) [inline Dockerfile]: as discussed on IRC, rename this to STI_LOCATION

**Comment by mfojtik** [comment]
LGTM, just one renaming suggestion

**Comment by soltysh** [comment]
Addressed @mfojtik [comment](https://github.com/openshift/ruby-20-centos/pull/7#discussion_r22587383).

**Comment by VojtechVitek** [comment]
LGTM

**Comment by mfojtik** [comment]
[test]

**Comment by bparees** [comment]
@mfojtik I assume the answer is no, but do we have a way to test this branch using your new STI tester logic?  ultimately that's probably what we want to enable those Jenkins jobs to do.

**Comment by bparees** [comment]
(lgtm, but if we can run @mfojtik 's test against it first, i'd like to do that)

**Comment by soltysh** [comment]
So far the test tag does not work (@mfojtik needs to talk to @danmcp regarding that issue). But in the meantime I've rebased against master and run those manually on my computer - works perfectly.

**Comment by danmcp** [comment]
@soltysh @mfojtik @bparees There is no [test]ing setup against this repo.

**Comment by bparees** [comment]
@danmcp @soltysh yeah i knew we didn't have the automatic testing here, I just mean to manually run it, possibly by configuring the jenkins STI image tester job to be able to pull from a branch.

**Comment by mfojtik** [comment]
@bparees I'm working with @danmcp do have the github pull builder plugin working in jenkins

**Comment by mfojtik** [comment]
[test]

**Comment by mfojtik** [comment]
[test]

**Pull request merged**